android/src/main/java/com/thebluealliance/androidclient/helpers/ThreadSafeFormatters.java

```diff
@@ -55,4 +55,8 @@ public static synchronized String formatDoubleOnePlace(double input) {
     public static synchronized String formatDoubleTwoPlaces(double input) {
         return DOUBLE_TWO_PLACES_FORMAT.format(input);
     }
+
+    public static synchronized String formatStat(double stat) {
```

Comment:
In case we want to change the formatting of just stats sometime in the future. I know that's unlikely, but I'd rather design for that now :stuck_out_tongue: